### PR TITLE
Release 4.4.6 GA

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Release 4.4.6 GA

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @lveyde @michalskrivanek @stoobie

### CODE REVIEW

File: source/download/alternate_downloads.md [hunk 1]
```diff
@@ -1,32 +1,35 @@
...
-For production installations, we recommend installing oVirt using the graphical Cockpit installer in the
+<img class="screenshot" src="download_1.png">
+
+For production installations, we recommend installing oVirt in the
 [Self-Hosted Engine configuration](/download). In this configuration, oVirt Engine and a Host are installed
 together with the Engine running as a Virtual Machine on that Host. This configuration is
 preferred because the Engine Virtual Machine will be highly available (once a second Host is added).
 
 However, if you prefer to run oVirt Engine standalone on physical hardware or another virtualization provider, you can install oVirt Engine
 and Nodes / Hosts separately.
```
Comment (stoobie) [inline source/download/alternate_downloads.md:17]: @sandrobonazzola You should choose one word (hosts or nodes) and stick with it.

Also whichever you choose, it shouldn't be capitalized.

Self-Host Engine should only be capitalized if it appears at the beginning of a sentence, and then it should be "Self-hosted engine"

File: source/download/alternate_downloads.md [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
 
-For production installations, we recommend installing oVirt using the graphical Cockpit installer in the
+<img class="screenshot" src="download_1.png">
+
+For production installations, we recommend installing oVirt in the
 [Self-Hosted Engine configuration](/download). In this configuration, oVirt Engine and a Host are installed
 together with the Engine running as a Virtual Machine on that Host. This configuration is
 preferred because the Engine Virtual Machine will be highly available (once a second Host is added).
 
 However, if you prefer to run oVirt Engine standalone on physical hardware or another virtualization provider, you can install oVirt Engine
```
Comment (stoobie) [inline source/download/alternate_downloads.md:16]: "...run *the* oVirt engine"

File: source/release/4.4.6/index.md [hunk 3]
```diff
@@ -703,7 +682,7 @@ In order to customize or disable the above default settings it is recommended to
 
 #### Contributors
 
-47 people contributed to this release:
+52 people contributed to this release:
```
Comment (stoobie) [inline source/release/4.4.6/index.md:685]: @sandrobonazzola What about the tech writers?

**Review by stoobie — CHANGES_REQUESTED** [review]

File: source/release/4.4.6/index.md (hunk 3 above)
Comment (sandrobonazzola) [reply source/release/4.4.6/index.md:685]: Would be happy to add them :-) Contributors list here is auto-generated from commit messages, adding manually

File: source/download/alternate_downloads.md (hunk 1 above)
Comment (sandrobonazzola) [reply source/download/alternate_downloads.md:17]: I guess we needs to sit and review the wording conventions across the whole website. Can be done in separate patch

File: source/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.adoc [hunk 4]
```diff
@@ -13,7 +13,7 @@ Hosts that can run the {engine-name} virtual machine are referred to as self-hos
 
 A storage domain dedicated to the {engine-name} virtual machine is referred to as the self-hosted engine storage domain. This storage domain is created by the installation script, so the underlying storage must be prepared before beginning the installation.
 
-See the link:{URL_downstream_virt_product_docs}html/planning_and_prerequisites_guide/[_Planning and Prerequisites Guide_] for information on environment options and recommended configuration. See link:{URL_downstream_virt_product_docs}html-single/planning_and_prerequisites_guide/#self-hosted-engine-recommendations[Self-Hosted Engine Recommendations] for configuration specific to a self-hosted engine environment.
+See the link:{URL_downstream_virt_product_docs}/planning_and_prerequisites_guide/[_Planning and Prerequisites Guide_] for information on environment options and recommended configuration. See link:{URL_downstream_virt_product_docs}/planning_and_prerequisites_guide/#self-hosted-engine-recommendations[Self-Hosted Engine Recommendations] for configuration specific to a self-hosted engine environment.
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line/index.adoc:16]: yes, tested working

File: source/release/4.4.6/index.md (hunk 3 above)
Comment (stoobie) [reply source/release/4.4.6/index.md]: Gotcha.

**Comment by stoobie** [comment]
LGTM

**Review by stoobie — APPROVED** [review]
LGTM

**Review by lveyde — APPROVED** [review]

**Pull request merged**